packages/cbioportal-frontend-commons/src/lib/oncokb/OncoKbUtils.ts

```diff
@@ -224,3 +237,39 @@ export function generateAnnotateStructuralVariantQuery(
         evidenceTypes: evidenceTypes,
     } as AnnotateStructuralVariantQuery;
 }
+
+export function generateAnnotateStructuralVariantQueryFromGenes(
```

Comment:
should this be generateAnnotatedStruct...  (see the "d" at end of annotated)

Reply:
and if so, seems like there are other places where you use this name

Reply:
`AnnotateStructuralVariantQuery` is the type. Do you still want me to change it?

Reply:
if what it means is "a structural varianty query that has been annotated" then yes.

Reply:
is it an API type?

Reply:
yes